src/core/components/App.tsx

```diff
@@ -30,7 +31,17 @@ const Loader = (): JSX.Element => (
 );
 
 const AppRoutes: React.FC = (): JSX.Element => {
+    const query = new URLSearchParams(useLocation().search);
+    const [clearError] = useMutation(CLEAR_ERROR);
     useTrackingHook();
+
+    useEffect(() => {
+        if (query.toString().length === 0) {
+            console.log('clear');
```

Comment:
Do we want to leave this console log here?

Reply:
No, that's a good catch

Reply:
Should now be resolved.